scripts/run-integration-tests.sh

```diff
@@ -227,6 +234,18 @@ echo "Updated!"
 CNI_IMAGE_UPDATE_DURATION=$((SECONDS - START))
 echo "TIMELINE: Updating CNI image took $CNI_IMAGE_UPDATE_DURATION seconds."
 
+if [[ $RUN_CALICO_TEST == true ]]; then
+    $KUBECTL_PATH apply -f "$TEST_CALICO_PATH"
+    sleep 5
+    while [[ $($KUBECTL_PATH describe ds calico-node -n=kube-system | grep "Available Pods: 0") ]]
```

Comment:
Do we want to wait infinitely till timeout?

Reply:
Yeah good idea, will add a timeout.